src/routes/dashboard/(draft)/ranks/+page.server.js

```diff
@@ -14,10 +14,8 @@ export async function load({ locals: { db, session }, parent }) {
     error(403);
 
   const { draft } = await parent();
-  const [availableLabs, rankings] = await Promise.all([
-    db.getLabRegistry(),
-    db.getStudentRankings(draft.id, user.id),
-  ]);
+  const rankings = await db.getStudentRankings(draft.id, user.id);
+  const availableLabs = await db.getLabRegistry();
```

Comment:
Curious about this change. Why did we serialize the query? There doesn't seem be any glaring dependencies here. If anything, I think we should wrap them in a `REPEATABLE READ` transaction.